chatbot/main.py

```diff
@@ -0,0 +1,63 @@
+## import libraries
+import os
+from vectordb import ChatbotAgent
+from load_data import load_data
+
+
+
+# main function

```

Comment:
Remove the useless comment. I see such kind of comments across many places.

Reply:
Thanks for the comment, done.